src/test/java/de/tum/in/www1/artemis/AbstractSpringIntegrationBambooBitbucketJiraTest.java

```diff
@@ -341,8 +341,8 @@ private void mockCloneAndEnableAllBuildPlans(ProgrammingExercise sourceExercise,
         }
         mockUpdatePlanRepository(exerciseToBeImported, SOLUTION.getName(), ASSIGNMENT_REPO_NAME, solutionRepoName, List.of());
         mockUpdatePlanRepository(exerciseToBeImported, SOLUTION.getName(), TEST_REPO_NAME, testsRepoName, List.of());
-        bambooRequestMockProvider.mockTriggerBuild(exerciseToBeImported.getProjectKey() + "-" + TEMPLATE.getName());
-        bambooRequestMockProvider.mockTriggerBuild(exerciseToBeImported.getProjectKey() + "-" + SOLUTION.getName());
+        bambooRequestMockProvider.mockTriggerBuild(projectKey + "-" + TEMPLATE.getName());
```

Comment:
This seems odd to me, if the testdata uses a different project key this would create a wrong value?

Reply:
Check where the value comes from:
`final var projectKey = exerciseToBeImported.getProjectKey();`

Reply:
Not sure why but I thought this was `"projectKey"` as a string.